src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageItemViewModel.cs

```diff
@@ -616,7 +616,7 @@ private async Task FetchIconAsync()
                     using (var memoryStream = new MemoryStream())
                     {
                         // Cannot call CopyToAsync as we'll get an InvalidOperationException due to CheckAccess() in next line.
-                        stream.CopyTo(memoryStream);
+                        await stream.CopyToAsync(memoryStream);
```

Comment:
There's a comment right above this line that says we shouldn't be calling CopyToAsync.